### DISCUSSION TITLE
compare diff missing

### DISCUSSION THREAD

**Discussion by hhc97** [community/community#6246] [Other Feature Feedback, Questions, & Ideas] [answered]
previously there was an UI element to show the diff of the changes between any two releases, this now seems to be missing, and I found it quite useful.

<img width="311" alt="image" src="https://user-images.githubusercontent.com/46637781/136972946-eb3ba5b9-9cc6-4b08-85cc-b56ac77f9a24.png">
This is the feature in question ^

**Comment by MylesBorins** [comment, marked as answer]
The compare box is there in the release view but was lost in the list view. I have made a feature request with engineering to get it added back

**Reply by MylesBorins** [reply]
We have an open PR to fix this that we can hopefully land on monday!

**Reply by MylesBorins** [reply]
This landed today, PTAL

**Reply by hhc97** [reply]
LGTM!

**Reply by hhc97** [reply]
btw, if GitHub is looking for a summer intern, hit me up :)
